website/docs/guides/explorer-qs.md

```diff
@@ -0,0 +1,137 @@
+
+ <Lightbox src="/img/quickstarts/dbt-cloud/dbt_explorer_dag.png" width="100%" title="dbt Explorer's lineage graph" />
+  
+- Use the [advanced search](/docs/collaborate/explore-projects#search-resources) feature to locate resources in your project. Perform hard searches and keyword searches. All resource names, column names, resources descriptions, warehouse relations, and code matching your search criteria will appear in the center of the page. Apply filters to fully refine your search
+- When searching for a column name, the results show all relational nodes containing that column in their schemas. 
+
+ <Lightbox src="/img/quickstarts/dbt-cloud/dbt_explorer_advanced_search.png" width="100%" title="dbt Explorer's advanced search feature" />
+
+### Hands-On
+1. On the left hand side, click on the company and ensure you're in the **Overview Project details** section → Next to **Overview** → Select **View Lineage**.
+
+2. Select a mart model and select everything downstream.
+   - Which models depend on models from another project?
+   - Click on the other project's lineage.
+
+3. Explore lenses and find table materializations in your current project.
+   - Which are the biggest view clusters?
+
+4. Experiment with different lenses, such as **Model layer**, **Test status**, **Resource type**, and so on.
+
+5. Drill down on a node and explore the **Column Lineage** and its evolution.
+   - Expand each column's full lineage.
+
+## Multi-project
+Use dbt Explorer to gain a deeper understanding of *all* of you dbt Cloud projects with its [multi-project capabilities](/docs/collaborate/explore-multiple-projects). 
+- See the number of public, protected, and private models, as well as metrics for each project.
+- View cross-project lineage and navigate between individual projects’ lineage graphs.
+- Explore column-level lineage across projects.
+
+### Hands-On
+- Identify which models in the project are public and being referenced by another project.
+- Switch between the lineage graphs of two different projects and explore the available assets.
+  - What sources are visible?
+- Use the multi-project column-level lineage to locate a column that exists in models across multiple projects.
+
+## Project recommendations
+Project recommendations suggests project improvements and provides insight into how you can create a better-documented, better-tested, and better-built dbt project.
+
+These recommendations are designed to build trust in your project and reduce confusion.
+
+To learn more about the specific suggestions and the reasons behind them, check out the [Project recommendations page](/docs/collaborate/project-recommendations).
+
+<Lightbox src="/img/quickstarts/dbt-cloud/dbt_explorer_project_recs.png" width="80%" title="dbt Explorer's project recommendation tab" />
+
+### Hands-On
+- Review your project recommendations and choose one improvement to implement (such as adding a description to a primary key that’s missing one).
```

Comment:
Do you want to leave this open ended to a the entire audience or guide them to a specific change?

If they are going to make a specific change, you should add the steps for making a change (go to the IDE, open a branch, etc. etc.).

Otherwise you leave yourself open to everyone trying something different and potential chaos